### PR TITLE
Create login page and form for user details

### PR DESCRIPTION
**Pull request by raniarusly**
landing page for when user is logged out 
image

user details form after user is logged in
image

### CODE REVIEW

File: src/main/webapp/form.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <div class="container">
+        <h1>StudentRecs</h1>
+        <h3>a recommendation system for students, by students!</h3>
+      </div>
+    </div>
+    <form name="user-details" action="/user" method="POST">
+      <p>Hello! my name is</p>
+      <input type="text" name="name" class="name">
+      <br><br>
+      <p>I am currently a student in department of</p>
```
Comment (lornaarmstrong) [inline src/main/webapp/form.html]: nit:

File: src/main/webapp/form.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  </head>
+  <body>
+    <div class="content">
+      <div class="container">
+        <h1>StudentRecs</h1>
+        <h3>a recommendation system for students, by students!</h3>
+      </div>
+    </div>
+    <form name="user-details" action="/user" method="POST">
+      <p>Hello! my name is</p>
```
Comment (lornaarmstrong) [inline src/main/webapp/form.html:19]: Are we collecting just first name? Or is this requesting their full name?

File: src/main/webapp/form.html (hunk 1 above)
Comment (lornaarmstrong) [inline src/main/webapp/form.html]: nit: this should be "I am currently a student in the department of"

**Review by lornaarmstrong — COMMENTED** [review]
Great work! I have a few questions about the login page, but looking good overall!

File: src/main/webapp/form.html (hunk 1 above)
Comment (raniarusly) [reply src/main/webapp/form.html]: thank you for the correction!

**Review by lornaarmstrong — APPROVED** [review]
LGTM! 

Note: can we use the convention of alphabetical css? (within each section {} in css, the properties are listed in alphabetical order). It makes it easier to see what's going on.

File: src/main/webapp/styles/form.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  right: 50%;
+  height: 100px;
+  width: 600px;
+  margin: -50px -300px;
+  text-align: center; 
+}
+
+.profilepic {
+  padding-left: 100px;
+}
```
Comment (MostafaGabriel) [inline src/main/webapp/styles/form.css:34]: nit: please add a new line :)

**Review by MostafaGabriel — APPROVED** [review]
LGTM! Thanks Rania :)

**Comment by MostafaGabriel** [comment]
> LGTM!
> 
> Note: can we use the convention of alphabetical css? (within each section {} in css, the properties are listed in alphabetical order). It makes it easier to see what's going on.

+1 Thanks Lorna!

**Pull request merged**